packages/algob/README.md

```diff
@@ -2,13 +2,29 @@
 
 Algob is the CLI (Command Line Interface) for Algo Builder. Think about it as an Ethereum Truffle but for Algorand projects.
 
+## Packages
+
+### `algob`
+
+This package provides a variety of high level tools like, `executeTransaction`, `mkTxParams`, etc. These high level tools help in making the development phase of algorand applications easier by acting as a middleware between the developer and `algosdk`.
+
+### `runtime`
```

Comment:
runtime package is put in different folder. `packages/runtime`.

Reply:
As this is the main docs page right now, so I added this here. In the task description too, it is mentioned to add this. It is just a short description